src/main/java/com/newrelic/telemetry/opentelemetry/export/NewRelicMetricExporter.java

```diff
@@ -32,6 +33,13 @@
   private final TelemetryClient telemetryClient;
   private final TimeTracker timeTracker;
   private final MetricPointAdapter metricPointAdapter;
+  private static final String implementationVersion;
+
+  static {
+    Package thisPackage = NewRelicMetricExporter.class.getPackage();
```

Comment:
this is ok for now, but we've found that this breaks when the code gets bundled inside of someone else's jar. Let's create an issue to track fixing it (like https://github.com/newrelic/newrelic-telemetry-sdk-java/issues/144)

Reply:
Ah good call, I removed the implementation version for now since it's optional!